ui/README.md

```diff
@@ -24,20 +24,19 @@
 
 This README outlines the details of collaborating on this Ember application.
 
-## Ember CLI Version Upgrade Matrix
```

Comment:
"Ember version" wasn't very clear since we have 3 different ember dependencies: `ember-source`, `ember-data` and `ember-cli` so I added separate columns and only included Vault versions where those versions changed. i looked into doing this dynamically but couldn't find a good solution since since we want to include historical versions, not just the current Vault ember versions

https://github.com/hashicorp/vault/blob/release/1.17.x/ui/package.json
https://github.com/hashicorp/vault/blob/release/1.15.x/ui/package.json
https://github.com/hashicorp/vault/blob/release/1.13.x/ui/package.json
https://github.com/hashicorp/vault/blob/release/1.11.x/ui/package.json
https://github.com/hashicorp/vault/blob/release/1.10.x/ui/package.json
https://github.com/hashicorp/vault/blob/release/1.9.x/ui/package.json